app/controllers/api_controller.rb

```diff
@@ -120,7 +120,7 @@ def set_access_control_headers
   # not have these. They would instead dealing with the /api/auth
   # mechanism.
   #
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
+  if Vmdb::Application.config.action_controller.allow_forgery_protection && respond_to?(:verify_authenticity_token)
```

Comment:
I thought protect_from_forgery had a :null_session setting or something for this purpose.  @matthewd might know better.